tests/suites/helpers.function

```diff
@@ -393,7 +393,7 @@ typedef enum
     do                                                            \
```

Comment:
Preexisting: for some reason, probably a bad merge or side-port from mbedtls to mbed-crypto, `ASSERT_ALLOC` is defined twice. If you need to change the macro, please remove one occurrence. Choose which one to make the crypto and tls copies of the file as close as possible.

Reply:
Fixed.